tests/core/BatchScheduler.spec.ts

```diff
@@ -2,12 +2,18 @@ import * as retry from 'async-retry';
 import BatchScheduler from '../../lib/core/BatchScheduler';
 import MockBatchWriter from '../mocks/MockBatchWriter';
 import MockBlockchain from '../mocks/MockBlockchain';
+import MockVersionedProvider from '../mocks/MockVersionManager';
```

Comment:
Nit:
```suggestion
import MockVersionManager from '../mocks/MockVersionManager';
```

Reply:
Good catch … I missed this during the name changes.